### ISSUE TITLE
When autologin after windows 10 updates is enabled, NVDA is not enabled at the logon screen

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#6327]
Since the Windows 10 anniversary update, there is a setting in windows update advanced settings to enable auto login after updates. When a pc is restarted due to installing updates, windows will automatically log on as the last user to apply the updates and to load the desktop, and than locks itself automatically. However, this is where it goes wrong, since NVDA does not read the log on screen at this stage. When you have to log on with speech support, you will have to enable narrator to access the logon screen. After log in was succesfull, Windows returns to the desktop and everything works as expected again.

I assume that this is a bug in Windows 10 anniversary update, but I report it here to investigate things as well as to monitor the status of the process to fix it.

**Comment by jcsteh** [comment]
Are you able to test whether this happens with Narrator if you set 
Narrator to start on the logon screen?

**Comment by leonardder** [comment]
Either that has to wait until a new update shows up, or I wil have to revert one and reinstall it. In any case, I"can test both with narrator and JAWS.

**Comment by leonardder** [comment]
I've been able to test already. Both JAWS and Narrator don;t talk
    either, so it definitely a Windows issue.

**Comment by josephsl** [comment]
Hi, I’ll do something about this so we can find out if it is really a Windows problem or NVDA should meet another dialog as its friend. Thankfully, I have a VM configured to use slow ring (Insider) and haven’t installed the latest cumulative update. Thanks.

**Comment by jcsteh** [comment]
Given that this also happens with Narrator, it's almost certainly not an NVDA issue. I'd say it has something to do with the Ease of Access framework. I'll try to reproduce at some point and file a bug with MS.

**Comment by leonardder** [comment]
The following is in the JAWS 2018 what's new:

> * When JAWS is configured to start at the login screen and after logon, addressed an issue in the Windows 10 Fall Creators' update where JAWS was not working as expected if the "Use my sign in info to automatically finish setting up my device after update or restart" option is enabled.

This suggest this has been solved by Microsoft somehow, or FS found a way to work around this.

**Comment by jcsteh** [comment]
I'm pretty sure (but not absolutely certain yet) that I've seen this
working of late and that this was fixed a while back. I certainly
experienced the bug previously, though. I'll try to take more notice next
time I get a major update and report back here.

**Comment by ehollig** [comment]
Any updates for this issue to anyone who has commented here? Have you noticed this? I do not believe I have.

**Comment by Adriani90** [comment]
@leonardder did you reproduce this issue during last months? Or can we close it?

**Comment by leonardder** [comment]
Nope, good catch. This seems to have been fixed.
